resin_release_tool/releaser.py

```diff
@@ -21,6 +23,69 @@ def get_canaries(self):
             if tag['tag_key'] == 'CANARY']
         return canaries
 
+    def get_tags_per_device(self):
```

Comment:
maybe it makes it clearer to have a short docstring here?

Reply:
maybe